### ISSUE TITLE
How to make a model dinamically point to a distinct database?

### ISSUE THREAD

**Issue by diegoaguilar** [balderdashy/sails#3058]
I'm creating a Sails application for which I got to do a panel administration for transactions to a model database. The point is, the backend architecture and data administration requires distinct databases for for same model in Sails application. 

So, my model looks like this:

    module.exports = {
    
      connection: 'seasonDB',
      autoPK: false,
    
      attributes: {
        id: {
          type: 'string',
          primaryKey: true,
          required: true
        },
        avatar: {
          type: 'string',
          defaultsTo: ''
        },
        price: {
          type: 'integer',
          defaultsTo: function () {
            var prices = [ 7000000, 5000000, 4000000, 2500000, 1000000 ];
            return prices[_.random(prices.length -1)];
          }
        },
        description: {
          type: 'string',
          defaultsTo: ''
        },
        status: {
          type: 'string',
          enum: ['suspended', 'inSquad','injured','out']
        }
      }
    };

Where `'seasonDB'` is defined at `config/connections.js`:

      seasonDB: {
        adapter: process.env.SEASON_DB_ADAPTER,
        host: process.env.SEASON_DB_HOST,
        port: process.env.SEASON_DB_PORT,
        user: process.env.SEASON_DB_USER,
        password: process.env.SEASON_DB_PASSWORD,
        database: process.env.SEASON_DB_DB
      }

So, is there any way to change the environmental variables and somehow reload the models with out reloading whole application?

**Comment by tjwebb** [comment]
You can reload the `orm` hook during runtime. Perhaps you can use this for inspiration: https://github.com/sgress454/sails-hook-autoreload

**Comment by diegoaguilar** [comment]
I've got not time to try it yet but I got one big doubt:

***How to introduce a change to the model or controller on the go?***

**Comment by sailsbot** [comment]
Thanks for posting, @diegoaguilar.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
